packages/playground/src/explorer/components/node_details_cards/twin_details_card.vue

```diff
@@ -17,38 +18,58 @@ export default {
   props: {
     node: {
       type: Object as PropType<GridNode>,
-      required: true,
+      required: false,
+    },
+    farm: {
+      type: Object as PropType<Farm>,
+      required: false,
     },
   },
 
   setup(props) {
     const loading = ref<boolean>(false);
     const twinFields = ref<NodeDetailsCard[]>();
+    const twin = ref<Twin>();
 
-    const mount = () => {
+    onMounted(async () => {
       loading.value = true;
+
+      if (props.farm) {
+        const { data } = await getTwins({
+          twinId: props.farm.twinId,
+        });
+        twin.value = data[0];
+      } else if (props.node) {
+        twin.value = props.node.twin;
+      }
       twinFields.value = getNodeTwinDetailsCard();
       loading.value = false;
-    };
-
-    onMounted(mount);
+    });
 
     const copy = (address: string) => {
       navigator.clipboard.writeText(address);
       createCustomToast("Copied!", ToastType.success);
     };
 
     const getNodeTwinDetailsCard = (): NodeDetailsCard[] => {
+      let twinId, accountId, relay;
+      if (twin.value) {
+        twinId = twin.value.twinId.toString();
+
+        accountId = twin.value.accountId;
+        relay = twin.value.relay;
+      }
+
       return [
-        { name: "ID", value: props.node.twin.twinId.toString() },
+        { name: "ID", value: twinId },
         {
           name: "Account ID",
-          value: props.node.twin.accountId,
+          value: accountId,
           icon: "mdi-content-copy",
           callback: copy,
           hint: "Copy the account id to the clipboard.",
         },
-        { name: "Relay", value: props.node.twin.relay },
+        { name: "Relay", value: relay },
```

Comment:
Can we set a default value as some values might return empty?

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/40770501/e68eab0f-bcc0-43c4-a801-56c0de3cc290)